sdk/communication/Azure.Communication.JobRouter/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## 1.0.0-beta.1 (2021-10-20)
+This is the beta release of Azure Communication Router .NET SDK. For more information, please see the [README][read_me].
```

Comment:
Neither of these links work due to them being enclosed in comments.

Reply:
The read_me link will be broken until the PR is merged to the main branch. Will be pushing in a second PR uncommenting the link section